src/scripts/run.sh

```diff
@@ -1,4 +1,6 @@
 #!/bin/bash
+apt-get update \
```

Comment:
Ik denk toch dat dit in de Dockerfile moet gebeuren. Het idee zoals de Dockerfile is opgezet is dat apt-get update  en install alleen in de builder wordt uitgevoerd. 

curl staat echter in /usr/bin op debian . Daarom moet ook in de Dockerfile bij runtime image worden toegevogd :

COPY --from=builder /usr/bin/ /usr/bin/

en ook libfiles voor curl (libcurl3)

dpkg -L curl 
/.
/usr
/usr/bin
/usr/bin/curl
/usr/share
/usr/share/doc
/usr/share/doc/curl
/usr/share/doc/curl/NEWS.Debian.gz
/usr/share/doc/curl/changelog.Debian.gz
/usr/share/doc/curl/changelog.gz
/usr/share/doc/curl/copyright
/usr/share/man
/usr/share/man/man1
/usr/share/man/man1/curl.1.gz
/usr/share/zsh
/usr/share/zsh/vendor-completions
/usr/share/zsh/vendor-completions/_curl
root@20f23f49883b:/# dpkg -L libcurl3
/.
/usr
/usr/lib
/usr/lib/x86_64-linux-gnu
/usr/lib/x86_64-linux-gnu/libcurl.so.4.4.0
/usr/share
/usr/share/doc
/usr/share/doc/libcurl3
/usr/share/doc/libcurl3/NEWS.Debian.gz
/usr/share/doc/libcurl3/changelog.Debian.gz
/usr/share/doc/libcurl3/changelog.gz
/usr/share/doc/libcurl3/copyright
/usr/share/lintian
/usr/share/lintian/overrides
/usr/share/lintian/overrides/libcurl3
/usr/lib/x86_64-linux-gnu/libcurl.so.3
/usr/lib/x86_64-linux-gnu/libcurl.so.4

De doc en man files zijn niet nodig, maar de lib files wel 
Het is wel netter als in de  runtime image geen apt-get nodig is

Reply:
Top, ik ga er naar kijken!